packages/amplication-client/src/App.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useEffect } from "react";
 import { Route, Switch, useHistory } from "react-router-dom";
+import { configure } from "react-hotkeys";
```

Comment:
It would be more clear with namespace import
```suggestion
import * as reactHotkeys from "react-hotkeys";
```